hivemq-edge/src/frontend/src/extensions/datahub/components/panels/ClientFilterPanel.spec.cy.tsx

```diff
@@ -0,0 +1,69 @@
+/// <reference types="cypress" />
+
+import { MockStoreWrapper } from '../../__test-utils__/MockStoreWrapper.tsx'
+import { DataHubNodeType } from '../../types.ts'
+import { getNodePayload } from '../../utils/node.utils.ts'
+import { ClientFilterPanel } from '../panels/ClientFilterPanel.tsx'
+import { Button } from '@chakra-ui/react'
+
+const wrapper: React.JSXElementConstructor<{ children: React.ReactNode }> = ({ children }) => (
+  <MockStoreWrapper
+    config={{
+      initialState: {
+        nodes: [
+          {
+            id: '3',
+            type: DataHubNodeType.CLIENT_FILTER,
+            position: { x: 0, y: 0 },
+            data: getNodePayload(DataHubNodeType.CLIENT_FILTER),
+          },
+        ],
+      },
+    }}
+  >
+    {children}
+    <Button variant="primary" type="submit" form="datahub-node-form">
+      SUBMIT{' '}
+    </Button>
+  </MockStoreWrapper>
+)
+
+describe('ClientFilterPanel', () => {
+  beforeEach(() => {
+    cy.viewport(800, 800)
+  })
+
+  it('should render the fields for a Validator', () => {
+    const onSubmit = cy.stub().as('onSubmit')
+
+    cy.mountWithProviders(<ClientFilterPanel selectedNode="3" onFormSubmit={onSubmit} />, { wrapper })
+
+    cy.get('h2').eq(0).should('contain.text', 'Client Filters')
+    // first item
+    cy.get('label#root_clients_0-label').should('contain.text', 'clients-0')
+    cy.get('label#root_clients_0-label + input').should('have.value', 'client10')
+    // first item
+    cy.get('label#root_clients_1-label').should('contain.text', 'clients-1')
+    cy.get('label#root_clients_1-label + input').should('have.value', 'client20')
+    // first item
```

Comment:
fun: I don't believe all of them can be first :D

```suggestion
    // second item
    cy.get('label#root_clients_1-label').should('contain.text', 'clients-1')
    cy.get('label#root_clients_1-label + input').should('have.value', 'client20')
    // third item
```